.github/workflows/build-android-apk.yaml

```diff
@@ -3,7 +3,8 @@ name: build-android-apk
 on:
   workflow_dispatch:
   push:
-    branches: [master]
+    branches:
+      - '**'
```

Comment:
Is it going to stay or it was for a testing reasons?

Reply:
Whoops

Reply:
It might use quite a bit of run time of mac os machines. Though we don't do a lot with RN repo.
I'm leaving the decision for you 😄